c_glib/test/run-test.rb

```diff
@@ -47,6 +47,11 @@ def initialize(data)
 rescue GObjectIntrospection::RepositoryError::TypelibNotFound
 end
 
+begin
+  Gandiva = GI.load("Gandiva")
+rescue GObjectIntrospection::RepositoryError::TypelibNotFound
+end
+
```

Comment:
It's better that we put this before `Parquet = GI.load("Parquet")` because we use the order in other places.

Reply:
I've fixed it.